tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationAttachmentNegativeTest.java

```diff
@@ -520,6 +520,23 @@ public void testInvalidConstAnnotElements() {
         validateError(compileResult, index, "expression is not a constant expression", line + 7, 16);
     }
 
+    public void testInvalidAnnotationAttachmentOnField() {
```

Comment:
Aren't we testing invalid annotation attachments on members of structured typed-binding patterns? I would rename accordingly.

Also, the bal function name is not updated.

Reply:
Updated